pkg/helper/options/options.go

```diff
@@ -0,0 +1,48 @@
+// Copyright © 2019 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package options
+
+import (
+	"bufio"
+	"fmt"
+	"strings"
+
+	"github.com/tektoncd/cli/pkg/cli"
+)
+
+type DeleteOptions struct {
+	Resource    string
+	ForceDelete bool
+}
+
+func CheckDeleteOptions(opts *DeleteOptions, s *cli.Stream, resoureName string) error {
```

Comment:
As a follow-up, we will need to hook this up to the rest of the `delete` commands :angel:

Reply:
yes, I wrote the issue https://github.com/tektoncd/cli/issues/390 for follow up :thumbsup: